lib/music-requests/MusicRequestsService.ts

```diff
@@ -149,4 +159,17 @@ export class MusicRequestsService {
       throw new UserMusicRequestNotFoundError();
     }
   }
+
+  private async checkMusicRequestsLimit(userId) {
+    const requestedArtist = this.getRequestedArtists(userId);
```

Comment:
What about 
```ts
const requestedArtist = await this.getRequestedArtists(userId);
const requestedAlbums = await this.getRequestedAlbums(userId);
const requestedSongs = await this.getRequestedSongs(userId);
```
?

So that you can omit the `await` here:
`(await requestedArtist).length + (await requestedAlbums).length + (await requestedSongs).length;`